.github/workflows/dart.yml

```diff
@@ -0,0 +1,44 @@
+name: Dart
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * *" # Every day at midnight
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    
+    strategy:
+      fail-fast: false
+      matrix:
+        sdk: [dev,beta,stable]
+        os: [ubuntu-latest, windows-latest, macos-latest]
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: cedx/setup-dart@v2
```

Comment:
Prefer `dart-lang/setup-dart@v0.1`

Note that that will use `channel` instead of `release-channel` for the parameter.

Reply:
Done